src/interface.ts

```diff
@@ -82,15 +130,15 @@ interface DesktopAgent {
   /**
    * Constructs a new intent
    */
-  intent(intent: IntentName, context: Context, target: String): Intent;
+  newIntent(intent: string, context: Context, target?: AppMetadata): Intent;
```

Comment:
I don't want to construct an intent before sending it 👎

Reply:
Agreed - see https://github.com/FDC3/API/issues/12

Reply:
I'm ok either way, I think the FDC3 intent is simply a string in some/most cases but an object in others (for example when you take into account friendly names and translations).  Happy to simplify but wondering if we are over simplifying (hard to tell without an implementation).

`deskAgent.resolveIntent( deskAgent.newIntent('viewChart', symbolContext) )`
vs 
`deskAgent.resolveIntent( 'viewChart', symbolContext)`

would be the rough savings.

Reply:
Agreed that there is not much difference between the two, apart from creating an object in the first case seems kind of unnecessary and wasteful, if that just resolves to the second case (especially if you are going to be doing that plenty of times). JavaScript is a functional language! :-D

Ultimately I guess I prefer the simplicity of the second case, it is _very_ easy to explain to people.